hcn/hcnutils_test.go

```diff
@@ -131,9 +131,6 @@ func CreateTestOverlayNetwork() (*HostComputeNetwork, error) {
 }
 
 func HcnCreateTestEndpoint(network *HostComputeNetwork) (*HostComputeEndpoint, error) {
-	if network == nil {
```

Comment:
😄 was it supposed to do something?

Reply:
it looks like it was empty when it merged 4 years ago
https://github.com/microsoft/hcsshim/commit/f670ba5ae4e2f8921d3cd1c6eb651abf7a1ed222#diff-0dde9f8620b7eab040e27b01223862b5aef3b2cca980bf42ed98f2933d006e9d